src/main/java/bio/terra/service/snapshot/SnapshotTableDao.java

```diff
@@ -79,7 +80,7 @@ private void createColumns(UUID tableId, Collection<Column> columns) {
                     .id(rs.getObject("id", UUID.class))
                     .table(table)
                     .name(rs.getString("name"))
-                    .type(rs.getString("type"))
+                    .type(TableDataType.valueOf(rs.getString("type")))
```

Comment:
I think we need to be careful here: I suspect that this may cause failures in older snapshots where the content of the type field is empty.  We could either:
a) handle that gracefully or
b) update the underlying DB to make sure that there are types

But the fact that there are perhaps empty types in the DB should be checked out in dev, alpha, and staging and fake prod (I think real prod is likely fine)

Reply:
Since we need to support snapshots forever,  and they're immutable, I'm guessing we go with A?
If empty type fields is the result of code from a byegone era, B sounds like a better option though, maybe that should be a separate ticket?

Reply:
Unfortunately, I'm not sure that this can be a separate ticket since this would likely make it impossible to read older snapshots out of the DB, so the get snapshots endpoint would fail if the page includes one of the older snapshots.  I'm going to check the dev and fake prod dbs to see what the situation is (se we're not talking in hypotheticals)